.github/workflows/mobile-test-run.yml

```diff
@@ -69,13 +69,13 @@ jobs:
       run: appium driver install uiautomator2
 
     - name: Start appium server
-      run: appium --base-path /wd/hub --relaxed-security --log-timestamp --log appium.log &
+      run: appium --relaxed-security --log-timestamp --log appium.log &
```

Comment:
what insecure features are we turning on (--relaxed-security)?

Reply:
@ikalinin1 ?